site/content/docs/getting-started/03-containers/index.md

```diff
@@ -0,0 +1,241 @@
++++
+title = "Containers"
+description = "Learn how to use container widgets."
+draft = false
+weight = 3
+template = "docs/page.html"
+slug = "containers"
+
+[extra]
+lead = "Now we're going to try out some built-in container widgets that will allow us to organize our content and build layouts."
+toc = true
+top = false
++++
+
+RAUI comes with a number of built-in container components that can be used to help group and lay out other components. You can find the list of core container types in the [API documentation][containers].
+
+[containers]: https://docs.rs/raui/latest/raui/core/widget/component/containers/index.html
+
+## Content Box
+
+The first container we'll look at is the [`content_box`]. Content boxes are simple containers that can hold multiple items, where each item's position is **not** effected by any of the other items in the box.
+
+A couple useful features of [`content_box`]s are that they can have a [transform][`contentboxprops::transform`] applied to them to scale, position, rotate, etc. the box and everything in it, and items inside them can specify [layout attributes][`contentboxitemlayout`] such as margin, alignment, and anchor.
```

Comment:
we should add a note that transform is used generally by the renderers only and it doesn't affect the layout areas produced by default layout engine, which means transforms are ignored also by interactions (pointer positions) - in other words: transforms are purely visual properties so user should be careful when setting them since they affect all the children visuals also.
i use them in Oxygengine to get effects like scaling or skewing of images, menu screens animated slide in/out, etc.

Reply:
Oh, OK. I didn't know that. Definitely a good tip to have in there! I'll update that.